src/lib/netlist/devices/nlid_system.h

```diff
@@ -74,7 +74,8 @@ namespace netlist::devices {
 				std::vector<pstring> inputs;
 				inputs.emplace_back("T");
 				m_vals.push_back(nlconst::zero());
-				for (int i=0; i < m_N(); i++)
+				m_I.reserve(m_N());
+				for (std::uint64_t i=0; i < m_N(); i++)
```

Comment:
Wouldn’t it make sense to `reserve(m_N() + 1)` on both `inputs` and `m_vals` before adding the first element to each of them above?  There are three vectors being filled in the loop here.

Reply:
Good catch. Done